src/react/workflow/__tests__/CreateWorkflow.test.tsx

```diff
@@ -0,0 +1,72 @@
+import CreateWorkflow from '../CreateWorkflow';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import {
+  mockOffsetSize,
+  reduxRender,
+  TEST_API_BASE_URL,
+} from '../../utils/test';
+import {
+  screen,
+  waitFor,
+} from '@testing-library/react';
+import { List } from 'immutable';
+import React from 'react';
+
+const instanceId = 'instanceId';
+const accountId = 'accountId';
+const accountName = 'pat';
+
+const server = setupServer(rest.post(
+  `${TEST_API_BASE_URL}/api/v1/instance/${instanceId}/accounts/${accountName}/workflows/create-workflow`,
+  (req, res, ctx) => res(ctx.json([])),
+),);
+
+beforeAll(() => {
+  server.listen({ onUnhandledRequest: 'error' });
+  mockOffsetSize(200, 800);
+  jest.setTimeout(10_000);
+});
+afterEach(() => server.resetHandlers());
+afterAll(() => server.close());
+
+describe('CreateWorkflow', () => {
+  it('should render a form for create workflow', async () => {
+    try {
+      reduxRender(<CreateWorkflow />, {
+        networkActivity: {
+          counter: 0,
+          messages: List.of(),
+        },
+        instances: {
+          selectedId: instanceId,
+        },
+        auth: {
+          config: { features: [] },
+          selectedAccount: { id: accountId },
+        },
+        configuration: {
+          latest: {
+           //locations,
+            endpoints: [],
+          },
+        },
+      });
+
+      await waitFor(() => screen.getByText(/Create New Workflow/i));
+
+      expect(screen.getByText('Rule Type')).toBeInTheDocument();
+
+      const createButton = screen.getByText('Create');
+      expect(createButton).toBeInTheDocument();
+
+      const cancelButton = screen.getByText('Cancel');
+      expect(cancelButton).toBeInTheDocument();
+
+    } catch (e) {
+      console.log('should render a form for create workflow: ', e);
+      throw e;
+    }
+  });
```

Comment:
This test is not enough, we want to interact with the rule type and ensure that when changing it the right form is rendered (expiration vs replicaiton)